src/Domain/ElementId.cpp

```diff
@@ -67,45 +69,45 @@ void ElementId<VolumeDim>::pup(PUP::er& p) noexcept {
 }
 
 template <size_t VolumeDim>
-std::ostream& operator<<(std::ostream& os, const ElementId<VolumeDim>& id) {
-  os << "[B" << id.block_id() << ',' << id.segment_ids() << ']';
+std::ostream& operator<<(std::ostream& os,
+                         const domain::ElementId<VolumeDim>& id) {
+  ::operator<<(os << "[B" << id.block_id() << ',', id.segment_ids()) << ']';
```

Comment:
ADL lookup means we it will only find the stream operator inside the enclosing namespace. In this case that's `std`, but we aren't allowed to put things into `std` so we need to specify where the operator lives.